runtime/sound/src/iltsound.cpp

```diff
@@ -356,15 +363,22 @@ ILTSoundSys* CFooSoundFactory::MakeSoundSystem( const char* pcSoundSystemName )
 #ifdef __LINUX
 
 DECLARE_SOUND_FACTORY(Linux)
+#ifdef USE_OPENAL
+ILTSoundSys* copenal_MakeSoundSys();
```

Comment:
this is already declared on line 249...
maybe also move csdl_MakeSoundSys up there

Reply:
Done. I kept csdl_MakeSoundSys where it is because it only belongs in the _LINUX #ifdef block.